core/src/Revolution/Processors/Security/Access/UserGroup/Context/GetList.php

```diff
@@ -127,7 +130,11 @@ public function prepareRow(xPDOObject $object)
         }
 
         $cls = '';
-        if (($objectArray['target'] === 'web' || $objectArray['target'] === 'mgr') && $objectArray['policy_name'] === 'Administrator' && ($this->userGroup && $this->userGroup->get('name') === 'Administrator')) {
+        if (
+            ($objectArray['target'] === 'web' || $objectArray['target'] === 'mgr')
+            && $objectArray['policy_name'] === 'Administrator'
+            && ($this->userGroup && $this->userGroup->get('name') === 'Administrator')
+        ) {
         } else {
```

Comment:
Is this if/else right? It looks like it was the same before, so I assume yes, but the empty body for the `if {}` looks weird.

Reply:
It is right, but perhaps this would read better (replacing the entire if else block):
```php
$cls .= ($objectArray['target'] === 'web' || $objectArray['target'] === 'mgr')
            && $objectArray['policy_name'] === 'Administrator'
            && ($this->userGroup && $this->userGroup->get('name') === 'Administrator')
            ? ''
            : 'pedit premove'
        ;
```
That said, I'd leave it for now as I have a different [proposed way](https://github.com/modxcms/revolution/pull/15919#issuecomment-980792211) of expressing the permissions that makes way more sense than sticking it in css class names, which would make this moot if accepted.